client/src/components/Admin/OrganizationEdit.js

```diff
@@ -53,10 +53,19 @@ const validationSchema = Yup.object().shape({
   hours: Yup.array().of(HourSchema),
   twitter: Yup.string()
     .matches(
-      /^https?:\/\/(www\.)?(twitter\.com|x\.com)\/.*/,
-      "Invalid URL, e.g. 'https://twitter.com/ or https://x.com/'"
+      /^@([a-zA-Z][a-zA-Z0-9_]{0,14}|[0-9][a-zA-Z0-9_]{0,14})$/,
```

Comment:
not sure if we want to save the `@` in the db. I tried going to a twitter profile with it and it redirects to the one without it 🤔

Reply:
yes, I believe both @username and username are accepted so I wasn't sure how people were going to write it. if you want without the @ symbol I can fix it to not accept it and only accept just the letters.